project/src/js/lichess/friends.ts

```diff
@@ -0,0 +1,88 @@
+class Friend {
```

Comment:
Prefer an interface here:
```
interface Friend {
   name: string
   playing: boolean
}

function makeUser(name: string, playing: boolean) {
   return { name, playing };
}
```

You should always use objects literals instead of classes, especially for simple needs.